File: c/average/2parcialAverage.c
//b) Cálculo de média de 10 números com resultados parciais

#include <stdio.h>

int main(){
    float avg, score, sum;

    for(int i = 0; i < 10; i++){
        printf("Por favor, informe a nota %d: ", i + 1);
        scanf("%f", &score);
        sum += score;
        avg = sum / (i + 1);
        
        if(i < 9){
            printf("Media parcial: %2.f\n", avg);
        }
    }
    printf("Media final: %2.f\n", avg);

    return 0;
}

File: python/2parcialAverage.py
# b) Cálculo de média de 10 números com resultados parciais

sum  = 0

for i in range(10):
    sum += float(input("Por favor, informe a nota {}: ".format(i + 1)))
    if(i < 9):
        print("A média parcial é: {}".format(sum/(i+1)))
else: 
    print("A média final é {}".format(sum/10))